src/providers/PageLoadProvider.tsx

```diff
@@ -40,6 +47,56 @@ const PageLoadProvider = ({ children }) => {
     if (isEmployeePage) setEntered(true)
   }, [isEmployeePage])
 
+  useEffect(() => {
```

Comment:
can you please move this `useEffect` to a new hook?
- suggestion: `useMintPoints`